project/Build.scala

```diff
@@ -226,7 +226,7 @@ object BijectionBuild extends Build {
 
   lazy val bijectionUtil = module("util").settings(
     osgiExportAll("com.twitter.bijection.twitter_util"),
-    libraryDependencies += "com.twitter" %% "util-core" % "6.20.0"
+    libraryDependencies += "com.twitter" %% "util-core" % "6.23.0"
```

Comment:
No thats ok, once its required to access the API's you need for the change the bump is fine